crates/biome_js_analyze/src/lint/nursery/use_import_extensions.rs

```diff
@@ -29,7 +29,7 @@ declare_rule! {
     /// import "./foo";
     /// ```
     /// ```js,expect_diagnostic,ignore
```

Comment:
I don't know how to fix diagnostics issue in the docs, likely issue with the docs generation and not the rule itself?

Reply:
You mean on the website?

Reply:
Yeah

Reply:
This is automatically generated in the website repository.

Reply:
I updated the repository, and now the checks are part of the CI